src/java/main/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -543,7 +546,7 @@ protected void pRequest2Txn(int type, long zxid, Request request,
                 }
                 
                 nodeRecord = getRecordForPath(ZooDefs.CONFIG_NODE);               
-                checkACL(zks, nodeRecord.acl, ZooDefs.Perms.WRITE, request.authInfo);                  
+                checkACL(zks, request.cnxn, nodeRecord.acl, ZooDefs.Perms.WRITE, request.authInfo, "/", null);
```

Comment:
`null` tells me that the method is not expecting a value from the caller in that position, which is better than an arbitrary value. I think it is fine to change it to `null`.